### PR TITLE
Add documentation about the HTTP API

### PR DESCRIPTION
**Pull request by saschagrunert**
This commit adds documentation about how to use the HTTP API and which
information it provides.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,33 @@ For more guidance in running `CRI-O`, visit our [tutorial page](tutorial.md)
 [libpod-hooks]: https://github.com/containers/libpod/blob/v0.6.2/pkg/hooks/README.md
 [spec-hooks]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config.md#posix-platform-hooks
 
+#### The HTTP API
+
+CRI-O exposes per default the [gRPC](https://grpc.io/) API to fulfill the
+Container Runtime Interface (CRI) of Kubernetes. Beside this, there exists an
```
Comment (vrothberg) [inline README.md]: s/Beside/Besides/

File: README.md [hunk 2]
```diff
@@ -131,6 +131,33 @@ For more guidance in running `CRI-O`, visit our [tutorial page](tutorial.md)
...
+additional HTTP API to retrieve further runtime status information about CRI-O.
+
+On a running CRI-O instance, we can access the API via a HTTP transfer tool like
```
Comment (vrothberg) [inline README.md]: s/via a/via an/

File: README.md (hunk 1 above)
Comment (saschagrunert) [reply README.md]: Thanks for the review :pray: Adapted as suggested.

File: README.md (hunk 2 above)
Comment (saschagrunert) [reply README.md]: Adapted as suggested.

File: README.md [hunk 3]
```diff
@@ -131,6 +131,33 @@ For more guidance in running `CRI-O`, visit our [tutorial page](tutorial.md)
 [libpod-hooks]: https://github.com/containers/libpod/blob/v0.6.2/pkg/hooks/README.md
 [spec-hooks]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config.md#posix-platform-hooks
 
+#### The HTTP API
```
Comment (mrunalp) [inline README.md]: We can probably add a statement saying that this isn't version and is likely to change.

Comment (saschagrunert) [reply README.md]: Yep, added an additional comment about that fact.

**Comment by haircommander** [comment]
LGTM

**Pull request merged**